File: exercicios/R/2029/2029.r
input = file('stdin','r')

while(TRUE){ 
    volume = as.double(readLines(input,n=1))
    diametro = as.double(readLines(input,n=1))

    v_pi = as.double(3.14156)

    raio = diametro/2

    area = v_pi * (raio*raio)

    altura = volume/area

    write(sprintf("ALTURA = %.2f",altura),'')
    write(sprintf("AREA = %.2f",area),'')
    if(length(volume) == 0){
        break;}
}

File: exercicios/JavaScript/2029/2029.js
var input = require("fs").readFileSync("stdin","utf-8");
var lines = input.split("\n");

/*
area = 3.14*diametro²

volume = area * altura



*/

while(lines.length > 0){
    var volume, diametro, area, altura, raio;
    
        volume = parseFloat(lines.shift()) 
        diametro = parseFloat(lines.shift());

        raio = diametro/2
        area = 3.14 * (raio * raio)
        altura = volume/area
        console.log(`ALTURA = ${parseFloat(altura).toFixed(2)}`)
        console.log(`AREA = ${parseFloat(area).toFixed(2)}`)
}